test/test_vmap.py

```diff
@@ -3547,6 +3546,38 @@ def test_randperm(self, device, randomness, use_generator):
             for i in range(B0):
                 assert torch.allclose(vmap_result[i], expected)
 
+    @parametrize('randomness', ['error', 'same', 'different'])
+    def test_feature_dropout(self, device, randomness):
+        # special case because functional feature dropout expects a batch dimension, so it doesn't match the
+        # pattern for other randomness
```

Comment:
should this just be `test_dropout` ? Or by "feature_dropout" do you mean `feature_alpha_dropout`?